vortex-ipc/src/chunked_reader/take_rows.rs

```diff
@@ -211,6 +207,7 @@ fn find_chunks(row_offsets: &Array, indices: &Array) -> VortexResult<Vec<ChunkIn
 }
 
 #[derive(Debug, Clone)]
+#[allow(dead_code)]
```

Comment:
?

Reply:
I removed the top level `#[allow(dead_code)]` and instead moved it here as this is the only part that has dead code now, this would be useful in coalesce chunks but that function is not that smart right now.